son/content/intermediate_outcomes/routes_into_work_(16_to_29_years)/destinations_following_the_end_of_compulsory_full-time_education/2.0.md

```diff
@@ -83,16 +77,70 @@ without a disability, across all socio-economic backgrounds.
 
 ---
 
+## Changes over time (new section)
```

Comment:
I guess we'll remove the words `(new section)` at some point?
Is that for Spence & Alex to re-word / should we remove it now / do SMC actually want the words "new section"?

Reply:
ah good spot! I've removed it, I think it was just a note for us :)